.github/workflows/terraformChecks.yml

```diff
@@ -82,4 +87,6 @@ jobs:
         run: make init-global
```

Comment:
If we decide the 'global` environment is no longer needed, then we can probably remove the sections referring to the `init-global`.

Reply:
Global is needed, as specified above and in the code.